roles/maintenance_15_rhel/tasks/main.yml

```
@@ -33,7 +33,9 @@
   vars:
     taskid: 15-011
     name: "Does yum check-update work"
-  ansible.builtin.command: "yum check-update -q"
+  ansible.builtin.command:
+    cmd: "yum check-update -q"
+    timeout: 300  # seconds
```

Where did you get that `timeout` parameter from? It's not in the documentation of the `ansible.builtin.command` module.

---

The pipeline also does not appear to recognize this parameter.

---

It's documented in the ansible.builtin.shell module I think, although it works here too

---

Ah, if I understand correctly that seems to be a [general task keyword](https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#task), so it would need to be on the same indentation level as the module name: 
```suggestion
  timeout: 300  # seconds
```